cmd/lotus-shed/dagspliter/dagsplitter.go

```diff
@@ -41,33 +42,36 @@ type Edge struct {
 }
 
 // Box contains part of a DAG, boxing of some edges
+// Wrapper around nodes to group into a fixed size. Alternative to actual
+// re-chunking.
 type Box struct {
 	// Nodes with external links
 	// root always 0-th; nodes don't have to depend on root; can be empty
-	Nodes []*BoxedNode
-
-	// Subgraphs of boxed nodes which fit here
-	// if no Nodes above, root is 0-th; can be empty
-	Internal []cid.Cid
+	// FIXME: What are the >1th nodes? Children? Maybe we could decouple that.
+	Nodes []cid.Cid
```

Comment:
Nodes which have both internal and external links (if they only had one of those, those nodes would be internal/external)

Reply:
Removed internal links.